modules/InfiniteStories/LoadMore.tsx

```diff
@@ -0,0 +1,17 @@
+interface Props {
+    isLoading: boolean;
+    onLoadMore: () => void;
+}
+
+export function LoadMore({ isLoading, onLoadMore }: Props) {
+    return (
+        <button
+            type="button"
+            onClick={onLoadMore}
+            disabled={isLoading}
+            className="block mx-auto my-8 py-3 px-4 border border-gray-300 rounded-md font-bold bg-gray-100 hover:bg-white hover:border-white hover:shadow-md active:shadow-inner"
```

Comment:
Okay, this prop is giving me eye twitch 😄

Reply:
I've been using `classnames` to organize the classes more cleanly in Privacy Portal, will do the same here.